### ISSUE TITLE
cp2p pre-trained model

### ISSUE THREAD

**Issue by luca-morreale** [pvnieo/DPFM#4]
Hello, is there any chance you can release the pre-trained models on cp2p?

There are the pre-trained models for shrec-cuts and shrec-holes, which is awesome!

Also, can you release the ground truth maps for test set of cp2p? 
Your evaluation code needs them, and it would be great to have to compute the error on the test set.

Thank you,
Luca
